### DISCUSSION TITLE
Changes in community license

### DISCUSSION THREAD

**Discussion by jac-dev** [ravendb/ravendb#15277] [General]
Hey

I recently came across a change in the licence agreement. The free community licence is all of a sudden limited to 50 licenses per customer. If I remember correctly, this wasn't the case earlier this year. It's a bit a bummer as we built our entire infrastructure around this license and this change hits us off guard.

From which version onwards does this change apply? Are there any other changes planned to the free community license which we should be aware of? Is it possible to get an even more reduced license without a quantity limit?

**Comment by ayende** [comment]
There are options available, yes. I would recommend that you would ping us at: sales@ravendb.net
